Signal/src/ProfileFetcherJob.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+
+@objc
+class ProfileFetcherJob: NSObject {
```

Comment:
To be clear, as a Swift neophyte my understanding is that the Swift-y thing to do is to not make classes NSObjects unless you know you're going to retain references to that class in Obj-C.